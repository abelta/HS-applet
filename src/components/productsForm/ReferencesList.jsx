import React from 'react';

const ReferencesList = () => <div></div>;

export default ReferencesList;
