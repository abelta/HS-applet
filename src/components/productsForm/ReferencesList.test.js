import React from 'react';
import renderer from 'react-test-renderer';
import ReferencesList from './ReferencesList';

describe('ReferencesList', () => {
  const data = [{ id: '1', name: 'name' }, { id: '2', name: 'name' }];
  const tree = renderer.create(<ReferencesList data={data} />).toJSON();

  it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});
