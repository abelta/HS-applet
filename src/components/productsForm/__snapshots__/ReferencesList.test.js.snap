// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReferencesList renders correctly 1`] = `
<div
  style={
    Object {
      "height": 200,
      "overflowY": "scroll",
    }
  }
>
  <ul
    className="collection"
  >
    <li
      className="collection-item row"
      style={
        Object {
          "padding": 0,
        }
      }
    >
      <button
        className="row btn-flat waves-effect waves-dark white"
        onClick={[Function]}
        style={
          Object {
            "border": "none",
            "margin": 0,
            "width": "100%",
          }
        }
      >
        <span
          className="col s2"
          style={
            Object {
              "fontWeight": "bold",
            }
          }
        >
          1
        </span>
        <span
          className="col s10"
        >
          name
        </span>
      </button>
    </li>
    <li
      className="collection-item row"
      style={
        Object {
          "padding": 0,
        }
      }
    >
      <button
        className="row btn-flat waves-effect waves-dark white"
        onClick={[Function]}
        style={
          Object {
            "border": "none",
            "margin": 0,
            "width": "100%",
          }
        }
      >
        <span
          className="col s2"
          style={
            Object {
              "fontWeight": "bold",
            }
          }
        >
          2
        </span>
        <span
          className="col s10"
        >
          name
        </span>
      </button>
    </li>
  </ul>
</div>
`;
